src/webdriver.nim

```diff
@@ -268,6 +274,18 @@ proc execute*(self: Session, code: string, args: varargs[JsonNode]): JsonNode =
 proc executeAsync*(self: Session, code: string, args: varargs[JsonNode]): JsonNode =
   self.internalExecute(code, args, "async")
 
+proc execute*(self: Session, code: string, args: varargs[Element]): JsonNode =
+  var newArgs = newSeq[JsonNode](args.len)
```

Comment:
Is there a better way to copy over the args?

Reply:
``@args`` might work

Reply:
Cool, done